docs/README.md

```diff
@@ -12,6 +12,7 @@
 - [Working with OAuth](#working-with-oauth)
 - [Frontend - Working with CSS and JS](#frontend---working-with-css-and-js)
 - [PHP Linting](#php-linting)
+- [Javascript Linting](#js-linting)
```

Comment:
Cool thing, thanks for adding the docs! Do you think these two sections should perhaps live under the same higher level heading "Linting", like such (see below)?

```markdown
## Linting

### PHP Linting

<!-- content... -->

### Javascript Linting

<!-- content... -->
```

Reply:
Yes, that makes sense. Thank you.